src/test/resources/org/jenkinsci/plugins/android_lint/parser/lint-results_r20.xml

```diff
@@ -17,7 +17,7 @@
     <issue
         id="Registered"
         severity="Warning"
-        message="The &lt;activity> MonitoredActivity is not registered in the manifest">
+        message="The &lt;activity&gt; MonitoredActivity is not registered in the manifest">
```

Comment:
Why would you alter the existing test data?

Please revert this and the other XML test data change.